### PR TITLE
default leafHost/leafPort to null

### PR DESCRIPTION
**Pull request by mmchen**

### CODE REVIEW

**Comment by mmchen** [comment]
@xhl1988 Could you please take a look? Thanks

**Comment by xhl1988** [comment]
@mmchen we can remove the migrator related code in https://github.com/uber/kafka-logger/blob/c0f2494cb53d0139b99613ba3e8f9722b93ad3e1/index.js#L114

**Comment by xhl1988** [comment]
@Raynos Could you please help take a look at this?

**Comment by mmchen** [comment]
@xhl1988 Done

File: index.js [hunk 1]
```diff
@@ -111,23 +111,23 @@ function KafkaLogger(options) {
...
-                if (options.blacklistMigrator) {
-                    kafkaRestClientOptions['blacklistMigrator'] = options.blacklistMigrator;
-                    kafkaRestClientOptions['blacklistMigratorUrl'] = options.blacklistMigratorUrl;
-                }
-            }
             if ('statsd' in options) {
                 kafkaRestClientOptions['statsd'] = options.statsd;
             }
+            if (options.batching) {
+                self.batching = options.batching;
```
Comment (xhl1988) [inline index.js]: My bad...
It should be kafkaRestClientOptions['batching'] = options.batching;

File: index.js [hunk 2]
```diff
@@ -111,23 +111,23 @@ function KafkaLogger(options) {
...
+            }
+            if (options.batchingWhitelist) {
+                self.batchingWhitelist = options.batchingWhitelist;
```
Comment (xhl1988) [inline index.js]: kafkaRestClientOptions['batchingWhitelist'] = options. batchingWhitelist;

**Comment by xhl1988** [comment]
lgtm

**Pull request merged**